content/departments/product-engineering/process/sdlc.md

```diff
@@ -2,33 +2,37 @@
 
 <span class="badge badge-note">SOC2/GN-98</span>
 
-Sourcegraph uses three types of approaches to drive changes:
+Sourcegraph uses three types of approaches to drive changes (either a new feature or extending an existing one).
 
 - [Product Documents](../product/process/product_documents.md) to communicate high-level product problems that need to be solved. All PDs are available in our [public Google Drive folder](https://drive.google.com/drive/folders/1UbuN9izpTj7ppJiduKI5tid8GEFuAiEx).
 - [Request For Comments](../../../company-info-and-process/communication/rfcs/index.md) to communicate technical problems or specific solutions in response to a Product Document.
 - Tickets (GitHub, GitHub discussions, JIRA, ...) and Pull Requests to communicate targeted technical problems or solutions, possibly (but not always) in response to RFCs or PDs.
 
-Sourcegraph follows an agile process, which means that the teams go through repeated iterations on a solution in order to refine results and adapt to changes discovered during each phase.
+Sourcegraph follows an agile process, which means that the teams go through repeated iterations on a solution in order to refine results and adapt to changes discovered during each phase. Learn more about the [product process](../product/process/index.md).
```

Comment:
Is this SDLC or PDLC? My impression was this is SDLC, but judging from the other changes it sounds like security wants a PDLC (product development lifecycle). Should we conflate the two?

Reply:
Alternatively, we could move this into the "Product Documents" bullet point

Reply:
I never heard so far about a PDLC, and the GN-98 mentions an SDLC, so I believe we should not conflate the two at this stage. I suppose you're referring to the SAST change? Maybe that should be moved to the continuous integration pipeline docs as we have a link toward it already here (well actually it's not linked, but I'll push a fix in a second)

Reply:
I'm referring to the change on this line (link to product process)

Reply:
We can move that to line 7 maybe to avoid conflating them?